checker/src/org/checkerframework/checker/index/upperbound/messages.properties

```diff
@@ -1,2 +1,3 @@
 ### Error messages for the Upper Bound Checker
 array.access.unsafe.high=Potentially unsafe array access: the index could be larger than the array's bound\nfound   : %s\nrequired: an integer less than %s's length (@LTLengthOf("%s"))
+array.access.unsafe.high.constant=Potentially unsafe array access: %s is larger than %s's known minimum length\nfound   : %s\nrequired: an array with a minimum length of at least %s
```

Comment:
The following is an example of this warning:
```
Constants.java:9: error: [array.access.unsafe.high.constant] Potentially unsafe array access: 3 is larger than arr's known minimum length
        int j = arr[3];
                    ^
  found   : @UnknownVal int @ArrayLen(3) []
  required: an array with a minimum length of at least 4
```

The other `array.access.unsafe` warnings are written in terms of what's wrong with the index, rather than the array.  It seems to be a more common error to get the index wrong, rather than the length on the array incorrect.  

What about something like the following?
```
Constants.java:9: error: [array.access.unsafe.high.constant] Potentially unsafe array access: 3 is larger than arr's known minimum length (@ArrayLen(3))
        int j = arr[3];
                    ^
  value of index :  3
  array length    : @ArrayLen(3)
1 error
```
@mernst might have a better suggestion.